src/crustacean/core.clj

```diff
@@ -89,7 +97,7 @@
   [[type opts] ]
   (let [schema (case type
                  :keyword `s/Keyword
-                 :string `s/Str
+                 :string `(s/maybe s/Str)
```

Comment:
Thought: we should be able to mark in the schema whether a field is nullable, ya know?